src/python/WMCore/Services/WMArchive/DataMap.py

```diff
@@ -300,12 +297,26 @@ def convertStepValue(stepValue):
 
     return stepValue
 
+def cleanStep(idict):
+    """
+    perform clean-up of PFNs attributes in place for given dictionary
+    """
+    for step in ['input', 'output']:
+        data = idict.get(step, {})
+        for key, values in data.items():
+            for elem in values:
+                for skip in ['pfn', 'InputPFN', 'OutputPFN', 'inputpfns']:
```

Comment:
Shouldn't this be defined in the WMARCHIVE_REMOVE_FIELD variable? Maybe not, because it's actually removing keys from the FJR, right? Still, this would be better to be defined at the top.

Reply:
Maybe move it up in the module under FWJR_REMOVE_STEP_FIELD?